vertical-pod-autoscaler/e2e/v1beta2/common.go

```diff
@@ -130,6 +130,17 @@ func NewHamsterDeploymentWithResources(f *framework.Framework, cpuQuantity, memo
 	return d
 }
 
+// NewHamsterDeploymentWithGuaranteedResources creates a simple hamster deployment with specific
+// resource requests for e2e test purposes.
```

Comment:
The description seems missing the quaranteed part.

Reply:
Done